packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -105,6 +109,15 @@ export const CallWithChatControlBar = (props: CallWithChatControlBarProps & Cont
     () => (!props.mobileView ? getDesktopEndCallButtonStyles(theme) : undefined),
     [props.mobileView, theme]
   );
+  /* @conditional-compile-remove(control-bar-button-injection) */
+  const customButtons = useMemo(
+    () =>
+      generateCustomButtons(
+        onFetchCustomButtonPropsTrampoline(options !== false ? options : undefined),
+        options !== false ? options?.displayType : undefined
+      ),
+    [options]
+  );
 
   // when options is false then we want to hide the whole control bar.
```

Comment:
Do we want to maintain this logic for the custom buttons? Say for example Contoso wants to put their own entire control bar in replacing the one we provide. If they set options to `false` then neither set wont be visible even if they provide their own custom buttons.

Reply:
Would love to have a deeper conversation about this with you through a short call.
If options is provided as false, how would they provide custom buttons?

Reply:
After discussing with Don. 
It makes sense to leave it as is. If Contoso choses to hide the ControlBar then it will also hide custom buttons. 
If Contoso would like to build out a ControlBar using only the CustomButtons then they will need to hide all other default buttons individually through the options.